src/user-event/press/__tests__/press.real-timers.test.tsx

```diff
@@ -0,0 +1,304 @@
+import * as React from 'react';
+import {
+  Pressable,
+  Text,
+  TextInput,
+  TouchableHighlight,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import { createEventLogger, getEventsName } from '../../../test-utils';
+import { render, screen } from '../../..';
+import { userEvent } from '../..';
+import * as WarnAboutRealTimers from '../utils/warnAboutRealTimers';
+
+const mockWarnAboutRealTimers = jest
+  .spyOn(WarnAboutRealTimers, 'warnAboutRealTimers')
+  .mockImplementation();
+
+describe('userEvent.press with real timers', () => {
+  beforeEach(() => {
+    jest.useRealTimers();
+  });
+
+  test('calls onPressIn, onPress and onPressOut prop of touchable', async () => {
+    const { events, logEvent } = createEventLogger();
+    const user = userEvent.setup();
+
+    render(
+      <Pressable
+        onPress={logEvent('press')}
+        onPressIn={logEvent('pressIn')}
+        onPressOut={logEvent('pressOut')}
+        testID="pressable"
+      />
+    );
+    await user.press(screen.getByTestId('pressable'));
+
+    expect(getEventsName(events)).toEqual(['pressIn', 'press', 'pressOut']);
```

Comment:
IMO some of the core tests should also test the event payload. Not all of the tests but just the core cases.

Reply:
I now test the events payload in one of the tests with fake timers